xdrip-get-entries.sh

```diff
@@ -53,6 +53,14 @@ main()
     fi
   fi
 
+  if [ -z  "$alternateBluetoothChannel" ]; then
```

Comment:
How would this ever not be an empty string?  Undocumented environment variable setting?

Reply:
Good point. I will remove that check